HTMLtask1/templStyle1.css

```diff
@@ -0,0 +1,311 @@
+html {
+margin: 0;
+padding: 0;
+font-family: 'Lato', sans-serif;
+}
+
+.main-page {
+  width: 100vw;
+  height: 100vh;
+  background-image: url("pics/mainBackground.jpg");
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+}
+
+.main {
+  width: 600px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#triangleM {
+  margin: 120px 0 60px 0;
+}
+
+.main-head {
+  text-align: center;
+  color: #ffffff;
+  font-size: 54px;
+  font-weight: 700;
+  text-transform: uppercase;
+  letter-spacing: 8.62px;
+}
+
+.main-par {
+  text-align: center;
+  color: #ffffff;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 29px;
+  text-transform: uppercase;
+  letter-spacing: 0.9px;
+  margin: 40px 0 50px 0;
+}
+
+#scroller {
+  margin: 43px 0 40px 0;
+}
+
+.about {
+  width: 100vw;
+  height: 60vh;
+  position: relative;
+}
+
+.about hr {
+  visibility: hidden;
+}
+
+.about-head {
+  position: absolute;
```

Comment:
don't use `position: absolute` in case you don't have `position:relative` in your parent element